File: Anul II/Medii de proiectare si programare/Laborator/lab03/Java/src/main/java/com/ubb/model/Race.java
package com.ubb.model;

public class Race extends Identifiable<Long> {
    private String name;
    private Integer engineCapacity;

    {
        setID(0L);
    }

    public Race(String name, Integer engineCapacity) {
        this.name = name;
        this.engineCapacity = engineCapacity;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getEngineCapacity() {
        return engineCapacity;
    }

    public void setEngineCapacity(Integer engineCapacity) {
        this.engineCapacity = engineCapacity;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab05/Ubb.BikeContest.Model/src/Race.cs
namespace Ubb.BikeContest.Model;

public class Race : Identifiable<long>
{
    public Race(string name, int engineCapacity)
    {
        Name = name;
        EngineCapacity = engineCapacity;
    }

    public string Name { get; set; }

    public int EngineCapacity { get; set; }
}